pkg/kubecost/bingen.go

```diff
@@ -1,5 +1,10 @@
 package kubecost
 
+// Default Version Set (uses -version flag passed) includes shared resources
+// @bingen:generate:Window
```

Comment:
Is there any danger of a version mismatch here in the future because asset and allocation both require `Window`?

Reply:
If you update window's version, assets and allocation will fail to load, and you'll basically get an error back from decoding causing a rebuild. Basically, each resource has it's own version.

Reply:
Killer :+1: